### PR TITLE
DX-1563 Add app manifest to example apps

### PR DESCRIPTION
**Pull request by janza**

### CODE REVIEW

**Review by mettin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,7 +15,15 @@ Make sure you visit our [developer documentation](https://developers.miro.com) t
...
 - For more information, visit our [developer documentation](https://developers.miro.com).
 
-### Miro Web SDK
+### App configuration
+
+All examples contain `app-manifest.yaml` file which can be used to quickly configure the example app.
+
+- Go to [App settings](https://miro.com/app/settings/user-profile/apps)
+- Click on Edit in manifest
+- Paste the yaml configuration and click Save
```
Comment (marcospinello) [inline README.md:24]:
```suggestion
- Go to [App settings](https://miro.com/app/settings/user-profile/apps).
- Click **Edit in manifest**.
- Paste the YAML configuration, and click **Save**.
```

File: README.md [hunk 2]
```diff
@@ -15,7 +15,15 @@ Make sure you visit our [developer documentation](https://developers.miro.com) t
   If you prefer, you can install and use equivalent alternatives, such as `yarn` or `pnpm`.
 - For more information, visit our [developer documentation](https://developers.miro.com).
 
-### Miro Web SDK
+### App configuration
+
+All examples contain `app-manifest.yaml` file which can be used to quickly configure the example app.
```
Comment (marcospinello) [inline README.md:20]:
```suggestion
All examples include an [`app-manifest.yaml` file](https://developers.miro.com/docs/app-manifest) to quickly configure the example app.
```

File: examples/asset-search/app-manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# See https://developers.miro.com/docs/app-manifest on how to use this
+appName: Asset Search
```
Comment (marcospinello) [inline examples/asset-search/app-manifest.yaml:2]:
```suggestion
appName: Miro Asset Search
```

File: examples/drag-and-drop/app-manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# See https://developers.miro.com/docs/app-manifest on how to use this
+appName: Drag and Drop
```
Comment (marcospinello) [inline examples/drag-and-drop/app-manifest.yaml:2]:
```suggestion
appName: Drag and drop
```

File: examples/node-stickies-csv/app-manifest.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# See https://developers.miro.com/docs/app-manifest on how to use this
+appName: CSV to Stickies
```
Comment (marcospinello) [inline examples/node-stickies-csv/app-manifest.yaml:2]:
```suggestion
appName: CSV to sticky notes
```

File: examples/stickynotes-to-shapes/app-manifest.yaml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# See https://developers.miro.com/docs/app-manifest on how to use this
+appName: Stickies to shapes
```
Comment (marcospinello) [inline examples/stickynotes-to-shapes/app-manifest.yaml:2]:
```suggestion
appName: Sticky notes to shapes
```

**Review by marcospinello — COMMENTED** [review]
I left some suggestions. You are welcome to accept or reject them!

**Pull request merged**